addons/web/static/src/js/views/kanban/kanban_column_progressbar.js

```diff
@@ -256,6 +253,29 @@ var KanbanColumnProgressBar = Widget.extend({
         });
     },
 
+    /**
+     * @private
+     * @param {string} value
+     */
+    _setActiveFilter(value) {
```

Comment:
```suggestion
    _applyActiveFilter(value) {
```

or maybe

```suggestion
    _toggleActiveFilter(value) {
```

or **_compute…** or **_prepare…**…